xla/backends/gpu/runtime/command_buffer_thunk.h

```diff
@@ -57,6 +57,8 @@ class CommandBufferThunk : public Thunk {
 
   void ForAllThunks(absl::FunctionRef<void(const Thunk*)> fn) const override;
 
+  std::string ToString(int indent) const override;
```

Comment:
It'd probably make sense to have a default value for indent here, so one could just call ToString().